### PR TITLE
{vis}[GCCcore/9.3.0] orca v1.3.1

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/o/orca/orca-1.3.1-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'Binary'
+
+name = 'orca'
```
Comment (lexming) [inline easybuild/easyconfigs/o/orca/orca-1.3.1-GCCcore-9.3.0.eb:6]: This name for the package will be very confusing with `ORCA`. I would use the name `plotly-orca` which is the same name used by the package in conda.

Comment (akesandgren) [reply easybuild/easyconfigs/o/orca/orca-1.3.1-GCCcore-9.3.0.eb:6]: Perhaps, but we already have an older version in the repo.

Comment (verdurin) [reply easybuild/easyconfigs/o/orca/orca-1.3.1-GCCcore-9.3.0.eb:6]: Yes, I just updated what was already there. Happy to proceed with the consensus view.

Comment (smoors) [reply easybuild/easyconfigs/o/orca/orca-1.3.1-GCCcore-9.3.0.eb:6]: my vote would be for renaming both the existing easyconfig and this one, as well as updating NanoComp-1.10.1-intel-2019b-Python-3.7.4.eb which depends on it. `plotly-orca` seems appropriate.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/94ed94d60a301bd6289412a218c329bc for a full test report.

**Review by lexming — APPROVED** [review]
I'll make another PR to fix the name of the package.

LGTM

**Comment by lexming** [comment]
Going in, thanks @verdurin !

File: easybuild/easyconfigs/o/orca/orca-1.3.1-GCCcore-9.3.0.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/o/orca/orca-1.3.1-GCCcore-9.3.0.eb:6]: Done in #12015

**Pull request merged**